net-im/mattermost-desktop-bin/mattermost-desktop-bin-4.6.2.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN%-*}"
+
+inherit desktop eutils
```

Comment:
*eutils.eclass* shouldn't be used for ebuilds anymore. I don't see any functions in this ebuild requiring it either.